### PR TITLE
New - Manage your compliance with Axway

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 7567c9a |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-306.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/manage_compliance/compliance_axwaycli.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+---
+Mange your API Compliance with the Axway Central CLI.
+
+## Before you start
+
+* You will need to have the Axway Central CLI installed and with a Central Administrator role.  Refer to https://docs.axway.com/bundle/amplify-central/page/docs/integrate_with_central/cli_central/index.html
+
+## Objectives
+
+* Using the Axway Central Central CLI:
```
Comment (jquick-axway) [inline content/en/docs/manage_compliance/compliance_axwaycli.md:14]: It says "Central" twice.

File: content/en/docs/manage_compliance/compliance_axwaycli.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    * Learn how to manually initiate a compliance lint job.
+    * Learn how to view the results of the compliance validation.
+
+### Step 1
+
+Enable Design and Security compliance validation for an environment.  When this is enabled, compliance validation will be initiated for all API services in the selected environment. 
+
+Example of viewing the environment resource for an envivronment named, test, with the get command:
+~~~
+axway centrea get environment test -o yaml
```
Comment (jquick-axway) [inline content/en/docs/manage_compliance/compliance_axwaycli.md:26]: Typo: "centrea" should be "central"

**Review by jquick-axway — CHANGES_REQUESTED** [review]

**Comment by robertengaxway** [comment]
> @robertengaxway , do you need help in providing example YAML/JSON? I gave you example files a few weeks earlier that should help with this.

Yes please

File: content/en/docs/manage_compliance/compliance_axwaycli.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+title: Manage your compliance with Axway Central CLI
+linkTitle: Manage your compliance with Axway Central CLI
+weight: 30
+---
+Mange your API Compliance with the Axway Central CLI.
+
+## Before you start
+
+* You will need to have the Axway Central CLI installed and with a Central Administrator role.  Refer to https://docs.axway.com/bundle/amplify-central/page/docs/integrate_with_central/cli_central/index.html
+
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/manage_compliance/compliance_axwaycli.md:12]: added it

File: content/en/docs/manage_compliance/compliance_axwaycli.md (hunk 2 above)
Comment (robertengaxway) [reply content/en/docs/manage_compliance/compliance_axwaycli.md:27]: fixed typo

**Comment by jquick-axway** [comment]
@lbadenhop, @robertengaxway, I don't have permission to make changes to this branch. So, I made the changes needed in my own forked branch here...
https://github.com/Axway/amplify-central/pull/323

**Comment by lbadenhop** [comment]
Closing PR. Replaced by https://github.com/Axway/amplify-central/pull/323.

**Pull request closed without merging**